Kickstarter-iOS/Views/Cells/DashboardActionCell.swift

```diff
@@ -45,6 +45,7 @@ internal final class DashboardActionCell: UITableViewCell, ValueCell {
     _ = self |> baseTableViewCellStyle()
     self.isAccessibilityElement = false
     self.accessibilityElements = [self.activityButton, self.messagesButton, self.postUpdateButton]
+      .map { $0 as Any }
```

Comment:
Should we use `.compactMap { $0 }` to explicitly state that we're dealing with optionals?

BTW: we're not calling `super.bindStyles()` (not sure if it's intentional here)

Reply:
Ah, sure I can change that to `compactMap`. I can add the super call... or perhaps a separate PR, don't really want to make any functional changes in this kind of PR.